### ISSUE TITLE
Loss 0 when finetuning

### ISSUE THREAD

**Issue by antran89** [facebook/C3D#75]
Hi,
When I tried to fine tune the model on UCF101 dataset with provided script, the loss is always 0
```
I0214 18:49:35.721410 17331 solver.cpp:237] Iteration 20, lr = 0.0001
I0214 18:49:35.805744 17331 solver.cpp:87] Iteration 20, loss = 0
I0214 18:50:40.039757 17331 solver.cpp:237] Iteration 40, lr = 0.0001
I0214 18:50:40.123868 17331 solver.cpp:87] Iteration 40, loss = 0
I0214 18:51:33.513945 17331 solver.cpp:237] Iteration 60, lr = 0.0001
I0214 18:51:33.599153 17331 solver.cpp:87] Iteration 60, loss = 0
I0214 18:52:36.863263 17331 solver.cpp:237] Iteration 80, 
```
Please give me some feedbacks.
Thanks

**Comment by antran89** [comment]
Sorry I forgot include label in the input txt file. Thanks
